lib/house.rb

```diff
@@ -0,0 +1,61 @@
+class House
+  attr_reader :price,
```

Comment:
Since you have a `price` method defined below, you should not include an attr_reader for price.